CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 * Changed: avoid saving empty mutations in Graph#saveElementMutations 
 * Added: support to read multiple StreamingPropertyValues with on query
 * Added: Graph.findPath exclusion labels
+* Added: Elasticsearch: Field query string support
```

Comment:
This should be moved to the 2.5.3 section of the changelog as 2.5.2 has been released.